packages/flutter/lib/src/material/chip.dart

```diff
@@ -262,6 +261,12 @@ abstract class SelectableChipAttributes {
   /// ```
   ValueChanged<bool> get onSelected;
 
+  /// Elevation to be applied on the chip during the press motion. 
```

Comment:
I'd say something like:
The z-coordinate (elevation) at which to place this material when the press gesture is active.
(and at the other site).